rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -49,14 +49,15 @@ def _decode_collateral_events(
             balance=Balance(),
             location_label=user,
             notes=f'{"Enable" if tx_log.topics[0] == ENABLE_COLLATERAL else "Disable"} {token.symbol} as collateral on AAVE v2',  # noqa: E501
-            counterparty=CPT_AAVE_V2,
+            protocol=CPT_AAVE_V2,
+            address=cast(ChecksumEvmAddress, transaction.to_address),  # transaction.to_address is not None here  # noqa: E501
```

Comment:
what is the case where this is None? Contracts deployments is one irc but not sure what other can have it

Reply:
No other case afaik. Only contracts deployment.